pages/demo/tinymce.tsx

```diff
@@ -0,0 +1,34 @@
+import { Editor } from '@tinymce/tinymce-react';
+import React, { useRef } from 'react';
+
+import { Layout } from '@/components/Layout';
+
+const TinymceEditor = () => {
+  const editorRef = useRef<Editor>(null);
+  const tinymcePlugins = ['link', 'lists', 'autoresize', 'image'];
+  const tinymceToolbar =
+    'blocks fontfamily |' +
+    'bold italic underline strikethrough forecolor backcolor |' +
+    'alignleft aligncenter alignright alignjustify |' +
+    'bullist numlist blockquote link image';
+
+  return (
+    <Layout>
+      <Editor
+        onInit={(_, editor) => (editorRef.current = editor)}
```

Comment:
_Cannot assign to 'current' because it is a read-only property._
위와 같은 에러 메시지가 뜨는데, [tinymce 공식 가이드](https://www.tiny.cloud/docs/integrations/react/)에서는 위와 같이 세팅하라고 하네요.